modules/table/src/main/java/org/apache/ignite/internal/table/KeyValueViewImpl.java

```diff
@@ -47,8 +47,8 @@
      * @param valueMapper Value class mapper.
      * @param tx The transaction.
      */
-    public KVViewImpl(InternalTable tbl, SchemaRegistry schemaReg, KeyMapper<K> keyMapper,
-        ValueMapper<V> valueMapper, @Nullable Transaction tx) {
+    public KeyValueViewImpl(InternalTable tbl, SchemaRegistry schemaReg, KeyMapper<K> keyMapper,
```

Comment:
`keyMapper` and `valueMapper` are unused. Why? Should this parameters be removed?

Reply:
They are useless for binary views.
These mappers describes how columns should be mapped to user POJO fields.